dsp/modules/googlevertexai.py

```diff
@@ -0,0 +1,174 @@
+"""Module for interacting with Google Vertex AI."""
+from typing import Any, Dict
+
+import backoff
+from pydantic_core import PydanticCustomError
+
+from dsp.modules.lm import LM
+
+try:
+    import vertexai  # type: ignore[import-untyped]
+    from vertexai.language_models import CodeGenerationModel, TextGenerationModel
+    from vertexai.preview.generative_models import GenerativeModel
+except ImportError:
+    print("Not loading VertexAI because it is not installed.")
```

Comment:
let's remove the print statement here to avoid printing every time someone imports dspy

Reply:
i do it